contracts/dev-utils/contracts/src/AssetData.sol

```diff
@@ -0,0 +1,32 @@
+/*
+
+  Copyright 2019 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.5.5;
+pragma experimental ABIEncoderV2;
+
+import "./LibAssetData.sol";
+
+
+// solhint-disable no-empty-blocks
+contract AssetData is
```

Comment:
Why do we need to have this contract? I can see that it's used in tests, but it doesn't seem like it's used in any Solidity contracts.

Reply:
It's important for us to expose EVM implementations of asset data encoding and decoding, in order for us to be able to easily support any language in the future.  For example, if someone wants to interact with our contracts from Java, they'd need to have a Java-native implementation of this functionality (which we don't want to provide, and which we don't want to burden them with having to write), OR, they could just call into our EVM implementation, and go on their merry way.  With this, any language that has a Web3 implementation can easily be used with our contracts.

Prior to this PR, the intention was to publish `DevUtils` as an interface to `LibAssetData`, among other things.  But given that we decided to split up the contracts, I needed a vehicle to deploy the library, so I created this contract.